Server-Setup-and-Maintenance-Linux.md

```diff
@@ -118,50 +118,31 @@ cd ../sql
 for f in *.sql
   do
      echo -n "Importing $f into the database..."
-     mysql xidb -u topazadmin -ptopazisawesome < $f && echo "Success"      
+     mysql xidb -u lsbadmin -plsbisawesome < $f && echo "Success"      
   done
 cd ..
 ```
-Now we are going to enter the username and password for sql inside three different files login.conf, map.conf, and search_server.conf
+Now we are going to enter the username and password for sql inside `network.lua`
 
-First, copy these config files to the conf folder:
+First, copy the .lua settings files from the `settings\default\` folder to the `settings\` folder:
 ```
-cp conf/default/* conf/
+cp settings/default/* settings/
 ```
 
-Then edit them as follows:
+Then edit the copied `network.lua` as follows:
 
 ```
-cd conf
-echo -e "\n#DB_VER: `git rev-parse --short=4 HEAD`" >> version.conf
-nano login.conf
+cd settings
+echo -e "\n#DB_VER: `git rev-parse --short=4 HEAD`" >> network.lua
```

Comment:
version was moved into login settings

https://github.com/LandSandBoat/server/blob/base/settings/default/login.lua#L15

Reply:
nvm misreading, saw "version.conf" and looked for it, not "db_ver" which is the databse tool thing